app/card_content.tsx

```diff
@@ -0,0 +1,74 @@
+interface ImageDetails {
+    title: string,
+    path: string,
+}
+
+interface Details {
+    title: string,
+    text: string,
+}
+
+interface StageLayoutCardContent {
+    image: ImageDetails,
+    summary: Details,
+    description: Details,
+    stack: Details,
+}
+
+interface StageCardContent {
+    projectName: string,
+    date: string,
+    content: StageLayoutCardContent
```

Comment:
nit: please add trailing comma for consistency. (personally, for interfaces I normally omit commas)

```suggestion
    content: StageLayoutCardContent,
```